components/ledger/ledger-utxo-token-cache/src/main/kotlin/net/corda/ledger/utxo/token/cache/services/PerformanceClaimStateStoreImpl.kt

```diff
@@ -69,9 +69,20 @@ class PerformanceClaimStateStoreImpl(
                 modifiedTime = clock.instant()
             )
 
-            val mismatchedState = stateManager.update(listOf(stateManagerState))
-                .map { it.value }
-                .firstOrNull()
+            val mismatchedState = try {
+               stateManager.update(listOf(stateManagerState))
+                    .map { it.value }
+                    .firstOrNull()
+            } finally {
```

Comment:
I think the idea was to "capture the exception"... whereas `finally` block will be called always.
```suggestion
            } catch(ex: Exception) {
                logger.warn("Exception during execution of an update", ex)
```